gsoc/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Comment:
likewise, can we avoid creating this file here, as it will be created in the other PR.